sdk/src/driftClient.ts

```diff
@@ -2076,54 +2082,98 @@ export class DriftClient {
 	): Promise<{ txSig: TransactionSignature; signedFillTx: Transaction }> {
 		const marketIndex = orderParams.marketIndex;
 		const orderId = userAccount.nextOrderId;
+		const bracketOrderIxs = [];
+
+		const placePerpOrderIx = await this.getPlacePerpOrderIx(orderParams);
 
-		const marketOrderTx = wrapInTx(
-			await this.getPlacePerpOrderIx(orderParams),
-			txParams?.computeUnits,
-			txParams?.computeUnitsPrice
-		);
 		for (const bracketOrderParams of bracketOrdersParams) {
-			marketOrderTx.add(await this.getPlacePerpOrderIx(bracketOrderParams));
+			const placeBracketOrderIx = await this.getPlacePerpOrderIx(
+				bracketOrderParams
+			);
+			bracketOrderIxs.push(placeBracketOrderIx);
 		}
 
-		const fillTx = wrapInTx(
-			await this.getFillPerpOrderIx(
-				userAccountPublicKey,
-				userAccount,
-				{
-					orderId,
-					marketIndex,
-				},
-				makerInfo,
-				referrerInfo
-			),
-			txParams?.computeUnits,
-			txParams?.computeUnitsPrice
+		const fillPerpOrderIx = await this.getFillPerpOrderIx(
+			userAccountPublicKey,
+			userAccount,
+			{
+				orderId,
+				marketIndex,
+			},
+			makerInfo,
+			referrerInfo
 		);
 
-		// Apply the latest blockhash to the txs so that we can sign before sending them
-		const currentBlockHash = (
-			await this.connection.getLatestBlockhash('finalized')
-		).blockhash;
-		marketOrderTx.recentBlockhash = currentBlockHash;
-		fillTx.recentBlockhash = currentBlockHash;
+		const lookupTableAccount = this.lookupTableAccount ?? await this.fetchMarketLookupTableAccount();
+
+		// use versioned transactions if there is a lookup table account
+		if (lookupTableAccount) {
```

Comment:
im thinking this function can have a `useLookupTable` param which the ui passes in based on the wallet being used?